[do-not-merge] Rework as Linked Data Formats WG

BigBlueHat opened the pull request:
On recent calls the WG has been interested in expressing the wider ecosystem
being created around JSON-LD. This is the beginnings of a rework of our current
charter to express that under the name "Linked Data Formats Working Group" per
recent discussions.

There's a good bit more editing to do yet, but this is a start. :grin:

Cheers,
:tophat:

pchampin committed:
setup new charter and GH page action

the new charter is currently a copy of the old one

pchampin committed:
update to latest version of the charter

pchampin committed:
mark charter as 'proposed', and update changelog

pchampin committed:
add REC-track deliverables CBOR-LD and YAML-LD

pchampin committed:
update w3c.json and add PR-preview

pchampin committed:
Update 2024/json-ld-wg/index.html

Co-authored-by: Anatoly Scherbakov <altaisoft@gmail.com>

pchampin committed:
fix copy-pasto

pchampin committed:
Update 2024/json-ld-wg/index.html

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

BigBlueHat committed:
Merge branch 'w3c:gh-pages' into json-ld-wg

pchampin committed:
Merge pull request #1 from json-ld/add-cborld-and-yamlld

add REC-track deliverables CBOR-LD and YAML-LD

pchampin committed:
add non-normative deliverable for language support

address #3

pchampin committed:
Merge pull request #4 from json-ld/bp-language-support-note

add non-normative deliverable for language support

pchampin committed:
update end-date and timeline

pchampin committed:
align with latest charter remplate

pchampin committed:
Merge pull request #5 from json-ld/update-end-date

update end-date and timeline

gkellogg committed:
Updates to extend the core recommendations for 1.2. (#6)

* Updates to extend the core recommendations for 1.2.
* Update 2024/json-ld-wg/index.html

---------

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

gkellogg committed:
Core scope (#8)

* Updates scope of work for core JSON-LD 1.2 specs.


---------

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

pchampin committed:
remove unused files

to avoid confusion with w3c/charter-drafts

pchampin committed:
update README

plehegar committed:
Use DRAFT

pchampin committed:
Merge pull request #9 from plehegar/draft

Use DRAFT

pchampin committed:
Update index.html

pchampin committed:
apply suggestions from @svgeesus

https://github.com/w3c/strategy/issues/494#issuecomment-2627852565

BigBlueHat committed:
Rename the WG to Linked Data Formats.

No other content has been changed. Just the name.

BigBlueHat committed:
Rewrite the background section.

pchampin commented:
Please note that this PR deletes a number of unrelated files; this is not intentional!
The reason is that the source repo ([json-ld/json-ld-wg-charter](https://github.com/json-ld/json-ld-wg-charter)) was a fork from charter-drafts, from which all other charters were removed for clarity.

@BigBlueHat I suggest we close this PR without merging, and make a fresh PR once the proposed changes are merged into [json-ld/json-ld-wg-charter](https://github.com/json-ld/json-ld-wg-charter).

BigBlueHat committed:
Revert "Rename the WG to Linked Data Formats."

This reverts commit ca9cf5744e2306b49d436b3d89a5195fdd30c5f3.

BigBlueHat committed:
Switch WG name back in intro.

BigBlueHat committed:
Change `expression` to `structure` in intro.

Co-authored-by: Pierre-Antoine Champin <github-100614@champin.net>

BigBlueHat committed:
Tweak Scope intro.

Mostly removing the web-scale/web-developer specificity as JSON-LD reaches
well beyond the browser-based Web to bots, IoT, big data, AI, and LLMs.

BigBlueHat committed:
Remove static list of PM issues.

Also add note about new features referencing the W3C process.

Based on @gkellogg suggestion in https://github.com/json-ld/json-ld-wg-charter/pull/10#issuecomment-3070114842

BigBlueHat committed:
Add version numbers; bump CR dates.

BigBlueHat committed:
Tweak Other Deliverables.

Dropped the multiple languages in JSON proposal.

BigBlueHat committed:
Bump timeline dates.

BigBlueHat committed:
Remove RDF Dataset Canonicalization & Hash WG.

BigBlueHat committed:
Rename RDF-star WG and update refs.

BigBlueHat committed:
Remove most CGs from liaison list.

BigBlueHat committed:
Reorder liaison groups.

Ordering by rough activity in relationship to this WG.

BigBlueHat committed:
Move Latest Publication down to avoid confusion.

plehegar commented:
This PR must not be merged indeed. Please only propose a change on your charter document and don't touch other items.
